blueprints/cloud-operations/network-dashboard/cloud-function/main.py

```diff
@@ -109,92 +112,97 @@ def monitoring_interval():
+        config["monitored_projects"] = []
+
+    # Gets projects and folders to be monitored
+    config["monitored_projects"] = get_monitored_projects_list(config)
+
+    # Keep the monitoring interval up2date during each run
+    config["monitoring_interval"] = monitoring_interval()
+
+    metrics_dict, limits_dict = metrics.create_metrics(
+        config["monitoring_project_link"])
+
+    # IP utilization subnet level metrics
+    subnets.get_subnets(config, metrics_dict)
+
+    # Asset inventory queries
+    gce_instance_dict = instances.get_gce_instance_dict(config)
+    l4_forwarding_rules_dict = ilb_fwrules.get_forwarding_rules_dict(
+        config, "L4")
+    l7_forwarding_rules_dict = ilb_fwrules.get_forwarding_rules_dict(
+        config, "L7")
+    subnet_range_dict = networks.get_subnet_ranges_dict(config)
+
+    # Per Network metrics
+    instances.get_gce_instances_data(config, metrics_dict, gce_instance_dict,
+                                     limits_dict['number_of_instances_limit'])
+    ilb_fwrules.get_forwarding_rules_data(
+        config, metrics_dict, l4_forwarding_rules_dict,
+        limits_dict['internal_forwarding_rules_l4_limit'], "L4")
+    ilb_fwrules.get_forwarding_rules_data(
+        config, metrics_dict, l7_forwarding_rules_dict,
+        limits_dict['internal_forwarding_rules_l7_limit'], "L7")
+    peerings.get_vpc_peering_data(config, metrics_dict,
+                                  limits_dict['number_of_vpc_peerings_limit'])
+    dynamic_routes_dict = routes.get_dynamic_routes(
+        config, metrics_dict, limits_dict['dynamic_routes_per_network_limit'])
+
+    # Per VPC peering group metrics
+    metrics.get_pgg_data(
+        config,
+        metrics_dict["metrics_per_peering_group"]["instance_per_peering_group"],
+        gce_instance_dict, config["limit_names"]["GCE_INSTANCES"],
+        limits_dict['number_of_instances_ppg_limit'])
+    metrics.get_pgg_data(
+        config, metrics_dict["metrics_per_peering_group"]
+        ["l4_forwarding_rules_per_peering_group"], l4_forwarding_rules_dict,
+        config["limit_names"]["L4"],
+        limits_dict['internal_forwarding_rules_l4_ppg_limit'])
+    metrics.get_pgg_data(
+        config, metrics_dict["metrics_per_peering_group"]
+        ["l7_forwarding_rules_per_peering_group"], l7_forwarding_rules_dict,
+        config["limit_names"]["L7"],
+        limits_dict['internal_forwarding_rules_l7_ppg_limit'])
+    metrics.get_pgg_data(
+        config, metrics_dict["metrics_per_peering_group"]
+        ["subnet_ranges_per_peering_group"], subnet_range_dict,
+        config["limit_names"]["SUBNET_RANGES"],
+        limits_dict['number_of_subnet_IP_ranges_ppg_limit'])
+    routes.get_dynamic_routes_ppg(
+        config, metrics_dict["metrics_per_peering_group"]
+        ["dynamic_routes_per_peering_group"], dynamic_routes_dict,
+        limits_dict['dynamic_routes_per_peering_group_limit'])
+
+    return 'Function executed successfully'
+
+if CFv2:
```

Comment:
I'd change this part with

```python
@functions_framework.http
def main_http(request):
  main(None, None)

if CFv2:
  main = functions_framework.http(http)

Reply:
sorry I think it will not work  main = functions_framework.http(http)
can you please see it again.

Reply:
Yeah, it's untested. Ignore, we can change it later